api/proto/common/common.proto

```diff
@@ -3,8 +3,47 @@ package common;
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common";
 
 message G1Commitment {
-	// The X coordinate of the KZG commitment. This is the raw byte representation of the field element.
-	bytes x = 1;
-	// The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
-	bytes y = 2;
+  // The X coordinate of the KZG commitment. This is the raw byte representation of the field element.
+  bytes x = 1;
+  // The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
+  bytes y = 2;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////////////
+// Messages for WIP RPCs. These are subject to change, do not consider these to be stable API. //
+/////////////////////////////////////////////////////////////////////////////////////////////////
+
+// A unique identifier for a blob.
+message BlobKey {
+  oneof key {
+    HeaderBlobKey header_blob_key = 1;
+    BatchBlobKey batch_blob_key = 2;
+  }
+}
+
+// Uniquely identifies a blob based on its blob header hash.
+message HeaderBlobKey {
+  bytes header_hash = 1;
+}
+
+// Uniquely identifies a blob based on its certificate hash and blob index.
+message BatchBlobKey {
+  bytes batch_header_hash = 1;
```

Comment:
It'd be more readable if we make the wording consistent with comment: either it's batch_header_hash, or certificate_hash

Reply:
Comment fixed. This was leftover cruft from a previous iteration where we were calling it something different.